pkg/apis/cluster/v1alpha2/machine_types.go

```diff
@@ -163,11 +163,40 @@ type MachineStatus struct {
 	// +optional
 	Phase *string `json:"phase,omitempty"`
```

Comment:
Might be good to add Enum validation here to go with the defined MachinePhases

Reply:
I assumed we decided during the proposal to avoid changing this to an Enum and leaving it free-form, unless I'm mistaken 🤔